flash/image/detection/model.py

```diff
@@ -75,3 +79,7 @@ def __init__(
     def _ci_benchmark_fn(self, history: List[Dict[str, Any]]) -> None:
         """This function is used only for debugging usage with CI."""
         # todo
+
+    def set_predict_kwargs(self, value):
+        """This function is used to update the kwargs used for the prediction step."""
+        self.adapter.predict_kwargs = value
```

Comment:
I think it would be cleaner to do this with properties. Something like:
```suggestion
    @property
    def predict_kwargs(self) -> Dict[str, Any]:
        """The kwargs used for the prediction step."""
        return self.adapter.predict_kwargs
        
    @predict_kwargs.setter
    def predict_kwargs(predict_kwargs: Dict[str, Any]):
        self.adapter.predict_kwargs = predict_kwargs
```

One day we should find a way for this sort of thing to exist on the adapter and automatically be attached here...

Reply:
Hi @ethanwharris I will do the changes, both CHANGELOG and the properties. For what concern how to do it automatically, I think that a way is that to automatically update the attribute __dict__ of the class, taking the properties from the adapter or something like that :)

Reply:
I have done the changes